qa/workunits/cephtool/test.sh

```diff
@@ -197,6 +197,12 @@ function ceph_watch_wait()
     fi
 }
 
+function test_mon_osd_modify_not_exsited_option()
+{
+  ceph daemon osd.0 config set aaaaa 1 | grep "no such an option"
+  ceph daemon mon.a config set aaaaa 1 | grep "no such an option"
```

Comment:
@liupan1111 i am not sure if this is a known issue, but neither do i think it's environmental related.  if this is not tracked by any ticket in our tracker, maybe we should fix it. i am fine either way. but it'd be easier if you can fix it in this very PR. otherwise we need to wait until the other one is merged.